app/controllers/subsidy/detail/step/detail.js

```diff
@@ -63,231 +54,8 @@ export default class SubsidyDetailStepDetailController extends Controller {
     return this.model.step;
   }
 
-  get isActiveStep() {
```

Comment:
Its weird that this code is being deleted. This getter is responsible for showing the correct message on the bottom of the subsidy-step specifying its state: 
- "previous steps need to be submitted first"
- "date has expired"
- "step has not yet been started"

This information is still relevant and should stay in this app

Reply:
I'm not sure if these messages are relevant in the subsidiedatabank as they are warnings/error messages meant for the user that wants to submit the step in loket. I had stripped those messages out previously, but if we want to show those warnings I can add them back in!

Reply:
You're right it can be useful to show it here, I added it back in